Client/Frontend/Settings/Clearables.swift

```diff
@@ -127,7 +127,10 @@ class CacheClearable: Clearable {
             URLCache.shared.removeAllCachedResponses()
             
             var err: Error?
-            for item in ["Caches", "Preferences", "Cookies", "WebKit"] {
+            /* Please note that this is a hack. If possible please use a public api.
```

Comment:
Please remove this comment. If you feel this is inadequate, file a ticket instead of leaving a comment. Link to the code, and this PR.